packages/react/package.json

```diff
@@ -18,19 +18,29 @@
     "report": "WITH_REPORT=true pnpm build"
   },
   "dependencies": {
-    "@logto/browser": "^0.1.0"
+    "@logto/browser": "^0.1.0",
+    "@silverhand/essentials": "^1.1.6",
+    "react": "^17.0.2",
```

Comment:
peerDependencies?

Reply:
ok

Reply:
We already have `react ^16.8.0` in the peerDependencies list